### ISSUE TITLE
Readthedocs example fails in pyproj

### ISSUE THREAD

**Issue by MrBitBucket** [fitnr/svgis#4]
```shell
$ uname -a
Linux minikat 5.9.3-arch1-1 #1 SMP PREEMPT Sun, 01 Nov 2020 12:58:59 +0000 x86_64 GNU/Linux

$ pip freeze
appdirs==1.4.4
attrs==20.3.0
certifi==2020.6.20
click==7.1.2
click-plugins==1.1.1
cligj==0.7.0
coverage==5.3
cssselect==1.1.0
distlib==0.3.1
filelock==3.0.12
Fiona==1.8.17
lxml==4.6.1
munch==2.5.0
packaging==20.4
pluggy==0.13.1
py==1.9.0
pyparsing==2.4.7
pyproj==3.0.0
Shapely==1.7.1
six==1.15.0
svgis==0.5.0
tinycss2==1.1.0
toml==0.10.2
tox==3.20.1
utm==0.6.0
virtualenv==20.1.0
webencodings==0.5.1```

$ python -c"import sys;print(sys.version)"
3.9.0 (default, Oct 16 2020, 23:11:00) 
[GCC 10.2.0]

$ cat europe-lakes 
svgis draw ne_110m_admin_0_countries.shp ne_110m_lakes.shp \
    --crs EPSG:102013 \
    --scale 1000 \
    --simplify 90 \
    --style '.ne_110m_admin_0_countries { fill: tan } .Germany { fill: purple }' \
    --style '.ne_110m_lakes { fill: #09d; stroke: none; }' \
    --class-fields name \
    --bounds -10 30 40 65 \
    -o out.svg

 $ ls ne_110m*
ne_110m_admin_0_countries.cpg  ne_110m_admin_0_countries.prj          ne_110m_admin_0_countries.shp  ne_110m_admin_0_countries.VERSION.txt
ne_110m_admin_0_countries.dbf  ne_110m_admin_0_countries.README.html  ne_110m_admin_0_countries.shx

$ source europe-lakes 
Traceback (most recent call last):
  File "/home/robin/devel/svgis/bin/svgis", line 8, in <module>
    sys.exit(main())
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/cli.py", line 185, in draw
    click.echo(svgis.map(layer, **kwargs).encode('utf-8'), file=output)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/svgis.py", line 67, in map
    drawing = SVGIS(
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/svgis.py", line 453, in compose
    members = [svg.group(**self.compose_file(f, bounds, scalar=scalar, **kwargs))
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/svgis.py", line 453, in <listcomp>
    members = [svg.group(**self.compose_file(f, bounds, scalar=scalar, **kwargs))
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/svgis.py", line 349, in compose_file
    self.update_projected_bounds(self.in_crs, self.out_crs, unprojected_bounds, padding)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/svgis.py", line 228, in update_projected_bounds
    projected = bounding.transform(bounds, in_crs=in_crs, out_crs=out_crs)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/svgis/bounding.py", line 128, in transform
    transformer = Transformer.from_crs(in_crs, out_crs, skip_equivalent=True, always_xy=True)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/pyproj/transformer.py", line 370, in from_crs
    CRS.from_user_input(crs_to),
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/pyproj/crs/crs.py", line 440, in from_user_input
    return CRS(value, **kwargs)
  File "/home/robin/devel/svgis/lib/python3.9/site-packages/pyproj/crs/crs.py", line 296, in __init__
    super().__init__(projstring)
  File "pyproj/_crs.pyx", line 2302, in pyproj._crs._CRS.__init__
pyproj.exceptions.CRSError: Invalid projection: +init=epsg:102013 +no_defs +type=crs: (Internal Proj Error: proj_create: crs not found)
```
I ran all the cli tests with make test-cli and they seemed to work. However, make test had some failures.

**Comment by fitnr** [comment]
The current version has some trouble with certain ESPG codes that are derive from ESRI data. Using a very similar projection should work:
```
    svgis draw ne_110m_admin_0_countries.shp \
    --bounds -10 30 40 60 \
    --project EPSG:3035 \
    --scale 5000 \
    --id-field name \
    --style ".ne_110m_admin_0_countries {fill: tan;} #Germany { fill: purple }" \
```

A beta fix for the projection issue is ready, I'm just going to write some additional tests before pushing a new version.

**Comment by fitnr** [comment]
Fixed in v0.5.1, which was just published.
